python/pyarrow/_parquet.pxd

```diff
@@ -133,7 +133,9 @@ cdef extern from "parquet/api/schema.h" namespace "parquet" nogil:
 
     enum ParquetVersion" parquet::ParquetVersion::type":
         ParquetVersion_V1" parquet::ParquetVersion::PARQUET_1_0"
-        ParquetVersion_V2" parquet::ParquetVersion::PARQUET_2_0"
+        ParquetVersion_V2_0" parquet::ParquetVersion::PARQUET_2_0"
+        ParquetVersion_V2_4" parquet::ParquetVersion::PARQUET_2_4"
+        ParquetVersion_V2_6" parquet::ParquetVersion::PARQUET_2_6"
```

Comment:
should latest be exposed?

Reply:
For now it's not being used in the PyArrow code. We can expose it when needed.

Reply:
I don't know. As you say, it may not be good practice. If people need features from a specific version, they should probably use that version?

Reply:
We can always add this later, in case there is demand for it / a good use case.